source/style-guide/information-types/prototypes/reference/server-params-style.txt

```diff
@@ -0,0 +1,129 @@
+.. _meta-reference-server-params:
+
+=======================
+Server Parameters Style
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Details to Document
+-------------------
+
+When documenting a new server parameter, include the following details 
+in addition to specific information about the new parameter:
+
+.. list-table::
+   :widths: 40 40
+   :header-rows: 1
+
+   * - Documentation Detail
+     - Necessity
+     - Description
+
+   * - mongod / mongos Support
+     - Required
+     - Whether the parameter is supported on ``mongod``, ``mongos``, or both.
+       Use ``|mongod-only|``, ``|mongos-only|``, or ``|both|`` as appropriate.
+
+   * - Version the Parameter was Added or Changed In
+     - Optional
+     - 
+       - For recent versions, include the version the parameter was added 
+         or changed in.
+       - For archived versions, delete the version added tag.
+
+   * - OS Availability
+     - Optional
+     - Some parameters are only available on certain operating systems. If the 
+       parameter is only supported on certain operating systems, specify which 
+       it's supported on.
+
+   * - Argument Type
+     - Required
+     - The type of argument the parameter expects: integer, string, boolean,
+       document, etc.
+
+   * - Default Value
+     - Required
+     - Specify the default value the parameter takes.
+   
+   * - Startup and Runtime Support
+     - Required
+     - For details, see :ref:`startup-runtime-support`.
+
+.. _startup-runtime-support:
+
+Startup and Runtime Support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Whether the parameter is supported at startup, runtime, or both.
```

Comment:
```suggestion
Specify whether the parameter is supported at startup, runtime, or both.
```
Suggestion: Perhaps this should be a complete sentence since it's a new subsection

Reply:
Moved and updated. Thanks!